### PR TITLE
feat: introduce logs insertion by batch

### PR DESCRIPTION
**Pull request by paul-nicolas**
BEWARE: batch size must stay at 1 until we have implemented the account based lock and CQRS for transactions, accounts and balances.

### CODE REVIEW

File: components/ledger/pkg/storage/sqlstorage/worker/worker.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+		workerJob: workerJob,
+
+		models:     nil,
+		modelsChan: make(chan []Model[MODEL], 1024),
+	}
+}
+
+// Run should be called in a goroutine
+func (w *Worker[MODEL]) Run(ctx context.Context) {
+	w.ctx = ctx
```
Comment (paul-nicolas) [inline components/ledger/pkg/storage/sqlstorage/worker/worker.go:40]: Let's not use it in the struct

File: components/ledger/go.mod [hunk 2]
```diff
@@ -35,6 +35,7 @@ require (
 	go.opentelemetry.io/otel/sdk v1.13.0
```
Comment (Misty0411) [inline components/ledger/go.mod:35]: [
```suggestion
 	go.opentelemetry.io/otel/sdk # v1.13.0

Comment (flemzord) [reply components/ledger/go.mod:33]: @Misty0411 Why this modification ?

**Review by gfyrag — APPROVED** [review]

File: components/ledger/go.mod (hunk 2 above)
Comment (Misty0411) [reply components/ledger/go.mod:33]: Its not me

**Pull request merged**